xml/System/ReadOnlySpan`1.xml

```diff
@@ -379,8 +379,8 @@ This method returns `default` when `array` is `null`.
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>A read-only reference to an object of type T that can be used for pinning, and is required to support the use of span within a fixed statement.</summary>
```

Comment:
```suggestion
        <summary>A read-only reference to an object of type <typeparamref>T</typeparamref> that can be used for pinning. It is required to support the use of a span within a <code>fixed</code> statement.</summary>
```

Reply:
@rpetrusha the build passed, but shouldn't we be using `<typeparamref name="T" />` instead? Or are both valid? I ask because this is the first time I see this format.

Reply:
Sorry, @carlossanlop. Yes, we should be using `<typeparmref name="T" />`. I'm not sure what I was thinking.

Reply:
no worries!